atlas/package.json

```diff
@@ -80,6 +80,7 @@
     "eslint-plugin-react-hooks": "^4.0.4",
     "faker": "^5.1.0",
     "fluent-ffmpeg": "^2.1.2",
+    "glider": "^0.1.0",
```

Comment:
This is a library for PostgreSQL, let's remove that and regenerate `yarn.lock`

Reply:
Thanks, removed it